src/util/Components/Forms/CheckList.tsx

```diff
@@ -21,13 +24,42 @@ const CheckList = (props: PropsType) => {
   if (!inactiveClass) inactiveClass = "btn-secondary active";
   if (!activeClass) activeClass = "btn-success active";
 
+  const selectAll = () => {
+    let newSet:string;
+    props.onClickSet(getSet(newSet, props.items));
+  };
+  const deselectAll = () => {
+    let newSet = new Set<string>();
+    props.onClickSet(newSet);
```

Comment:
```suggestion
    props.onClickSet(new Set<string>());
```
Initialize the empty set inside method call.

Reply:
For this case, you can write code for selectAll and deselectAll methods inside the onClick handler. I suggest doing that instead of creating separate methods.